distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/AbstractIntegrationTest.java

```diff
@@ -672,200 +539,66 @@ protected Option installStartupFile(InputStream resourceInputStream, String dest
-        catalogBundle.setFanout(fanoutEnabled);
+    protected AdminConfig getAdminConfig() {
+        return adminConfig;
     }
 
-    /**
-     * @deprecated since 2.8.0. Use {@link ServiceManager#waitForRequiredBundles(String)
-     * getServiceManager.waitForRequiredBundles()} instead.
-     */
-    @Deprecated
-    protected void waitForRequiredBundles(String symbolicNamePrefix) throws InterruptedException {
-        serviceManager.waitForRequiredBundles(symbolicNamePrefix);
+    protected ServiceManager getServiceManager() {
+        return serviceManager;
     }
 
-    /**
-     * @deprecated since 2.8.0. Use {@link ServiceManager#waitForSourcesToBeAvailable(String, String...)
-     * getServiceManager.waitForSourcesToBeAvailable()} instead.
-     */
-    @Deprecated
-    protected void waitForSourcesToBeAvailable(String... sources) throws InterruptedException {
-        serviceManager.waitForSourcesToBeAvailable(REST_PATH.getUrl(), sources);
+    protected CatalogBundle getCatalogBundle() {
+        return catalogBundle;
     }
 
-    /**
-     * @deprecated since 2.8.0. Use {@link CatalogBundle#waitForFederatedSource(String)
-     * getCatalogBundle.waitForFederatedSource()} instead.
-     */
-    @Deprecated
-    protected FederatedSource waitForFederatedSource(String id)
-            throws InterruptedException, InvalidSyntaxException {
-        return catalogBundle.waitForFederatedSource(id);
+    protected SecurityPolicyConfigurator getSecurityPolicy() {
+        return securityPolicy;
     }
 
-    /**
-     * @deprecated since 2.8.0. Use {@link CatalogBundle#waitForCatalogProvider()
-     * getCatalogBundle.waitForCatalogProvider()} instead.
-     */
-    @Deprecated
-    protected CatalogProvider waitForCatalogProvider() throws InterruptedException {
-        return catalogBundle.waitForCatalogProvider();
+    protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
+        return urlResourceReaderConfigurator;
     }
 
-    /**
-     * @deprecated since 2.8.0. Use {@link CatalogBundle#getCatalogFramework()
-     * getCatalogBundle.getCatalogFramework()} instead.
-     */
-    @Deprecated
-    protected CatalogFramework getCatalogFramework() throws InterruptedException {
-        return catalogBundle.getCatalogFramework();
+    protected Integer getBasePort() {
+        return Integer.parseInt(System.getProperty(BASE_PORT.getSystemProperty()));
     }
 
-    /**
-     * @deprecated since 2.8.0. Use {@link ServiceManager#getMetatypeDefaults(String, String)
-     * getServiceManager.getMetatypeDefaults()} instead.
-     */
-    @Deprecated
-    protected Map<String, Object> getMetatypeDefaults(String symbolicName, String factoryPid) {
-        return serviceManager.getMetatypeDefaults(symbolicName, factoryPid);
+    public static InputStream getFileContentAsStream(String filePath) {
+        return AbstractIntegrationTest.class.getClassLoader().getResourceAsStream(filePath);
```

Comment:
This will load resources relative to AbstractIntegrationTest. If other classes in different bundles extending AbstractIntegrationTest would like to load resources relative to their class, they should call their own classloader.